### DISCUSSION TITLE
I can't make this DXF fit

### DISCUSSION THREAD

**Discussion by AhmetPolat43** [OpenOrienteering/mapper#1803] [Questions & Answers] [answered]
Hello, I have a question again. I have a dxf file in my hand but it has no coordinates. I can't get that dxf fit on my coordinate satellite image.
![Adsız](https://user-images.githubusercontent.com/62382116/102363334-5fe74a80-3fc6-11eb-8fdf-7a3136ae984b.jpg)

**Comment by AhmetPolat43** [comment]
I can't move the DXF file wherever I want so

**Comment by dg0yt** [comment, marked as answer]
There is no tool to fit your **- Map -** on your satellite image. These tools are meant to fit templates like TIFF images or DXF data on your map.

The DXF file is not in your list of templates. This means you do not currently use the DXF file as a template. I guess you opened like a regular (map) file, or you imported into a new map.

To start with the DXF file as a template, and import it after positioning, proceed like this:
1. Create a new, empty map with the desired scale and symbol set.
2. Add some properly georeferenced template. This will guide you into a step where you define the map's georeferencing. Change the coordinate reference systems (CRS) to your preferred local standard, or choose UTM, or just leave the default. 
  If you don't have any other properly georeferenced template, you may just export the desired region from www.openstreetmap.org, giving you a `map.osm` file.
3. Add more templates (images etc.) as you like.
4. Now add the DXF file as a template. It won't be georeferenced, so you can adjust its scaling and position with the buttons you already tried.
   *Update:* Always select Coordinate system: "Real" until #1768 is resolved. Select "Position track at view center" unless you are sure that the DXF files really uses the map's coordinate reference system.
5. Once satisfied, you can turn the objects from the template into map objects by clicking the template's Edit button and selecting "Import and remove". This does similar things like File > Import..., however Mapper knows the desired positioning.

**Comment by dg0yt** [comment]
Converting this issue to a discussion, category "Question and answers"...

**Comment by AhmetPolat43** [comment]
How can I tell you? I have a DXF file with no coordinates. I want to fit it on my satellite view with coordinates, so I will have set the coordinate of my DXF file. And I'm looking for a way to move my uncoordinated DXF file.

**Comment by pkturner** [comment]
Before you can move a DXF, it must be opened as a template.
The DXF file is not in your list of templates. Did you try to add the DXF file as a template?

**Comment by AhmetPolat43** [comment]
Yes, I tried opening it as a template but I cannot edit the DXF file as if it was an image. I cannot convert it to any symbol.

**Reply by dg0yt** [reply]
How far did you follow the instructions I gave? 
Note the level of detail in my step-by-step instructions. If  you could state your approach in a similar way, we might offer even better help.

**Reply by jmacura** [reply]
This is intentional. You can not edit the file when opened as a template.
**First**, you must align the DXF template to your georeferenced TIFF imagery.
**Then**, you can import it as vector features. As @dg0yt already wrote: _by clicking the template's Edit button and selecting "Import and remove"._

Hope that helps.

**Comment by AhmetPolat43** [comment]
I have managed to get DXF coordinates right now. I'm sure it's correct. It fits in the same place as the satellite picture, but when I open it in Mapper, they go to completely different places and I can't find them. The last photo is as you said, I did it again.





![1](https://user-images.githubusercontent.com/62382116/102407401-21b94d80-3ffd-11eb-8269-042cabd007fa.jpg)
![2](https://user-images.githubusercontent.com/62382116/102407415-267e0180-3ffd-11eb-8019-87614cb8e807.jpg)
![3](https://user-images.githubusercontent.com/62382116/102407426-2aaa1f00-3ffd-11eb-8245-957f4e803d0c.jpg)
![4](https://user-images.githubusercontent.com/62382116/102407432-2c73e280-3ffd-11eb-9d96-6c1d114133f1.jpg)
![5](https://user-images.githubusercontent.com/62382116/102407453-339af080-3ffd-11eb-8990-8b4451cf2f87.jpg)
![6](https://user-images.githubusercontent.com/62382116/102407459-35fd4a80-3ffd-11eb-8410-7d2dd2cd28a6.jpg)
![7](https://user-images.githubusercontent.com/62382116/102407466-385fa480-3ffd-11eb-807c-3da558390c42.jpg)
![8](https://user-images.githubusercontent.com/62382116/102407479-3c8bc200-3ffd-11eb-9c2b-60281f9e8903.jpg)
![9](https://user-images.githubusercontent.com/62382116/102407485-401f4900-3ffd-11eb-98d4-6d601e18530c.jpg)
![10](https://user-images.githubusercontent.com/62382116/102407489-41507600-3ffd-11eb-822d-67a02618f1c0.jpg)

**Reply by dg0yt** [reply]
With the TIF template in your screen, in the dialog labeled "Track scaling and positioning", try choosing "Position track a view center".

(If your map's coordinate reference system is e.g. UTM, real coordinates are as high as 5000000. But your DXF may have coordinates close to zero. Then, "Positioning the track at given coordinates" will move it far away from the TIF template. In the worst case, Mapper will not be able to fit both templates into the screen even at "Show whole map").

**Comment by AhmetPolat43** [comment]
The strange thing is. I tried it about 3 times in one of them, the Mapper and DXF and Satellite view came to the same place, but what I do now does not work, I'll try again tomorrow.

**Reply by AhmetPolat43** [reply]
solved the problem thanks.

**Reply by dg0yt** [reply]
I am glad to hear that. There are some know issues in related steps (#1768, #1561), and that is why it is really important talk about steps in a reproducible way. I need to understand if you actually discovered a new bug ;-)

**Reply by AhmetPolat43** [reply]
My suggestion is to DXF in Turkey
The plan is very crowded and when I open it with the openorienteering mapper, it turns into direct lines. It is more practical to delete the unnecessary if we leave this to us.For example 👍 

![image](https://user-images.githubusercontent.com/62382116/102456334-088ebc00-4052-11eb-87c3-6d563e34b424.png)
